build.cmd

```diff
@@ -703,29 +701,41 @@ if "%BUILD_PROTO_WITH_CORECLR_LKG%" == "1" (
   popd
 )
 
+echo ---------------------------- Prepare FsLex and FsYacc -----------------------------
+
+echo %_msbuildexe% %msbuildflags% src\buildtools\FsLexYacc.proj /t:Restore
```

Comment:
These should only be used in the buildfromsource path - my understanding is (was?) that we use the nuget package elsewhere?